gb-ppu/src/registers/ppu_registers.rs

```diff
@@ -113,10 +116,13 @@ where
                 Ok(())
             }
             #[cfg(feature = "cgb")]
-            Bcpd | Bcps | Ocpd | Ocps => {
-                log::warn!("missing ppu BgObjPalettes registers write");
-                Ok(())
-            }
+            Bcpd | Bcps | Ocpd | Ocps => match self.lcd.try_borrow_mut() {
```

Comment:
Could be merged with Vbk line 106. The warning could log the name of the register when failing to write/read.